README.md

```diff
@@ -39,6 +44,12 @@ Be sure you have the correct Terraform version (0.13.0+), you can choose the bin
 
 - https://releases.hashicorp.com/terraform/
 
+## Example
+
+You can find Terraform examples in
+- examples/resources
```

Comment:
I'm not sure we need to mention `examples` folder here.

That folder is required by `github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs` which is generating documentation that is here https://registry.terraform.io/providers/indykite/indykite/latest/docs/resources/application_agent_credential

So maybe would be better to pinpoint the documentation than folder itself

Reply:
True, our docs.indykite.com documentation links the example folder as examples, that why I added it, but I can see to update both